payments-core/src/main/java/com/stripe/android/model/PaymentMethod.kt

```diff
@@ -1017,8 +1018,22 @@ constructor(
          *
          * [us_bank_account.routingNumber](https://stripe.com/docs/api/payment_methods/object#payment_method_object-us_bank_account-routing_number)
          */
-        @JvmField val routingNumber: String?
+        @JvmField val routingNumber: String?,
     ) : TypeData() {
+        /**
+         * The token of the Linked Account used to create the payment method
+         *
+         * [us_bank_account.linkedAccount](https://stripe.com/docs/api/payment_methods/object#payment_method_object-us_bank_account-linked_account)
+         */
+        @Deprecated(
+            message = "Renamed to 'financialConnectionsAccount', " +
+                "'linkedAccount' will be removed in a future major update",
+            replaceWith = ReplaceWith(expression = "financialConnectionsAccount")
+        )
+        @IgnoredOnParcel
```

Comment:
Nit: Can avoid this if we make it a computed property.

Reply:
Sadly it breaks the API as computed properties cannot be JVM fields.

Reply:
Oh. Feel free to merge then.